docs/docs-content/enterprise-version/install-palette/install-on-kubernetes/airgap-install.md

```diff
@@ -0,0 +1,881 @@
+  must set the `ingress.enabled` parameter to `false` in the **values.yaml** file.
+
+- A custom domain and the ability to update Domain Name System (DNS) records. You will need this to enable HTTPS
+  encryption for Palette.
+
+- If you are installing Palette behind a network proxy server, ensure you have the Certificate Authority (CA)
+  certificate file in the base64 format. You will need this to enable Palette to communicate with the network proxy
+  server.
+
+- Access to the Palette Helm Charts. Refer to the [Access Palette](../../enterprise-version.md#access-palette) for
+  instructions on how to request access to the Helm Chart
+
+<br />
+
+:::warning
+
+Do not use a Palette-managed Kubernetes cluster when installing Palette. Palette-managed clusters contain the Palette
+agent and Palette-created Kubernetes resources that will interfere with the installation of Palette.
+
+:::
+
+## Install Palette
+
+The following instructions are written agnostic to the Kubernetes distribution you are using. Depending on the
+underlying infrastructure provider and your Kubernetes distribution, you may need to modify the instructions to match
+your environment. Reach out to our support team if you need assistance.
+
+1.  Open a terminal session and navigate to the directory where you downloaded the Palette install zip file provided by
+    our support. Unzip the file to a directory named **palette-install**.
+
+    ```shell
+    unzip release-*.zip -d palette-install
+    ```
+
+2.  Navigate to the release folder inside the **vertex-install** directory.
+
+    ```shell
+    cd palette-install/charts/release-*
+    ```
+
+3.  Open the file **extras/cert-manager/values.yaml** in a text editor and append the URL to your OCI registry that also
+    includes the namespace or project that is hosting the Spectro Cloud images. The URL should be in the format
+    `<registry-url>/<namespace>`. In the example configuration below, the value `my-oci-registry.com/spectro-images` is
+    prefixed to each URL. Save the file after you have appended the URL.
+
+    ```yaml hideClipboard
+    image:
+      cainjectorImage: "my-oci-registry.com/spectro-images/gcr.io/spectro-images-public/release-fips/jetstack/cert-manager-cainjector:spectro-v1.11.0-20230427"
+      controllerImage: "my-oci-registry.com/spectro-images/gcr.io/spectro-images-public/release-fips/jetstack/cert-manager-controller:spectro-v1.11.0-20230427"
+      webhookImage: "my-oci-registry.com/spectro-images/gcr.io/spectro-images-public/release-fips/jetstack/cert-manager-webhook:spectro-v1.11.0-20230808"
+      amceResolverImage: "my-oci-registry.com/spectro-images/gcr.io/spectro-images-public/release-fips/jetstack/cert-manager-acmesolver:spectro-v1.11.0-20230427"
+    ```
+
+4.  Install Cert Manager using the following command. Replace the actual file name of the Cert Manager Helm Chart with
```

Comment:
Both Step 4 and Step 5 should be listed as a pre-requisite instead of being part of the install flow. It would provide more clarity than listing them as part of the installation process.

Reply:
No, users struggle with this so we will cover it